tests/TimeSeries/RollingReturn.test.ts

```diff
@@ -0,0 +1,54 @@
+import * as Assert from 'node:assert/strict';
+import * as MC from '../../code/connectors-morningstar.src';
+import { isNumber } from 'highcharts';
+
+export async function rollingReturnLoad (
+    api: MC.Shared.MorningstarAPIOptions
+) {
+    const connector = new MC.TimeSeriesConnector({
+        api,
+        currencyId: 'EUR',
+        endDate: '2020-01-31',
+        securities: [{
+            id: 'F0GBR04S23',
+            idType: 'MSID'
+        }],
+        series: {
+            type: 'RollingReturn',
+            rollingPeriod: 15
+        },
+        startDate: '2020-01-01'
+    });
+
+    Assert.ok(
+        connector instanceof MC.TimeSeriesConnector,
+        'Connector should be instance of TimeSeries class.'
+    );
+
+    Assert.ok(
+        connector.converter instanceof
+        MC.TimeSeriesConverters.RollingReturnSeriesConverter,
+        'Converter should be instance of TimeSeries RollingReturnSeriesConverter.'
+    );
+
+    await connector.load();
+
+    Assert.deepStrictEqual(
+        connector.table.getColumnNames(),
+        ['Date', connector.options.securities?.[0].id],
```

Comment:
I don't feel good with this kind of compare value for the test, I feel like hardcoding it would be better than `connector.options.securities?.[0].id`, and definitely more readable.